### ISSUE TITLE
emulator speed

### ISSUE THREAD

**Issue by mromanuk** [NardJ/esp8266-C64#2]
Is fast enough to play games? (using it directly with video)

**Comment by NardJ** [comment]
Yes it emulates the 6502 at a bit above 1Mhz (1.2Mhz is achievable). the C64 runs the 6502 CPU at 1.0Mhz.  The current implementation uses a browser as a display. The refreshing rate of my browser implementation is a limiting factor for games. However I know for sure a more adept HMTL5 programmer should be able to increase the refreshing rate enough for simple games. (If you also want to display video/cvbs on a tv using the esp8266, the emulation-rate will drop too much I expect.)

If you are able to improve the browser implementation, let me know!
